src/EnergyPlus/DataDaylighting.hh

```diff
@@ -53,7 +53,6 @@
 #include <ObjexxFCL/Array2D.hh>
 #include <ObjexxFCL/Array3D.hh>
 #include <ObjexxFCL/Array4D.hh>
-#include <ObjexxFCL/Array5D.hh>
```

Comment:
I don't see Array5D used anywhere else so that should be removed. I see the description saying it was not removed so another branch could pull out this unused code.